WeakAuras/WeakAuras.lua

```diff
@@ -78,6 +78,48 @@ function WeakAuras.InternalVersion()
   return internalVersion;
 end
 
+do
+  local currentErrorHandlerId
+  local currentErrorHandlerUid
+  local currentErrorHandlerContext
+  local function waErrorHandler(errorMessage)
+    local prefix = ""
+    local data
+    if currentErrorHandlerId then
+      data = WeakAuras.GetData(currentErrorHandlerId)
+    elseif currentErrorHandlerUid then
+      data = Private.GetDataByUID(currentErrorHandlerUid)
+    end
+    if data then
+      Private.AuraWarnings.UpdateWarning(data.uid, "LuaError", "error",
+        L["This aura has caused a lua error."] .. "\n" .. L["Install the addons bugsack and buggraber for detailed error logs."])
```

Comment:
L["This aura has caused a Lua error."] .. "\n" .. L["Install the addons BugSack and BugGrabber for detailed error logs."])